packages/hoppscotch-app/src/components/environments/teams/index.vue

```diff
@@ -178,4 +181,14 @@ const getErrorMessage = (err: GQLError<string>) => {
     }
   }
 }
+
+defineActionHandler("modals.team.environment.edit", (args?: any[]) => {
+  const [envName, variableName]: any = args
+  editingVariableName.value = variableName
+  const teamEnvToEdit: TeamEnvironment | undefined =
+    props.teamEnvironments.find((environment: TeamEnvironment) => {
```

Comment:
Same as above, the type of `environment` is automatically inferrable, hence you can drop the type definition.

```suggestion
    props.teamEnvironments.find((environment) => {
```